snet_cli/arguments.py

```diff
@@ -4,95 +4,113 @@
 import sys
 from pathlib import Path
 
-from snet_cli.commands import IdentityCommand, SessionSetCommand, SessionShowCommand, NetworkCommand, ContractCommand, OrganizationCommand, VersionCommand
+from snet_cli.commands import IdentityCommand, SessionSetCommand, SessionShowCommand, NetworkCommand, ContractCommand, \
+    OrganizationCommand, VersionCommand
 from snet_cli.identity import get_identity_types
 from snet_cli.utils import type_converter, get_contract_def
 from snet_cli.mpe_account_command import MPEAccountCommand
 from snet_cli.mpe_service_command import MPEServiceCommand
 from snet_cli.mpe_channel_command import MPEChannelCommand
-from snet_cli.mpe_client_command  import MPEClientCommand
+from snet_cli.mpe_client_command import MPEClientCommand
 from snet_cli.mpe_treasurer_command import MPETreasurerCommand
 from snet_cli.sdk_command import SDKCommand
 from snet_cli.utils_agi2cogs import stragi2cogs
 from snet_cli.config import get_session_keys, get_session_network_keys_removable
 
+
 class CustomParser(argparse.ArgumentParser):
     def __init__(self, default_choice=None, *args, **kwargs):
         self.default_choice = default_choice
         super().__init__(*args, **kwargs)
-
+    
     def error(self, message):
         sys.stderr.write("error: {}\n\n".format(message))
         self.print_help(sys.stderr)
         sys.exit(2)
-
+    
     def _parse_known_args(self, arg_strings, *args, **kwargs):
         if self.default_choice and not len(list(filter(lambda option: option in arg_strings, {'-h', '--help'}))):
             for action in list(filter(
-                lambda subparser_action: isinstance(subparser_action, argparse._SubParsersAction),
-                self._subparsers._actions
+                    lambda subparser_action: isinstance(subparser_action, argparse._SubParsersAction),
+                    self._subparsers._actions
             )):
                 if not len(list(filter(lambda arg: arg in action._name_parser_map.keys(), arg_strings))):
                     arg_strings = [self.default_choice] + arg_strings
-
+        
```

Comment:
I would remove such white-spaces in empty string (```sed -i 's/[ \t]*$//' arguments.py```), but of course it is very minor..

Reply:
Done.